test/controllers/PartialControllerSpec.scala

```diff
@@ -16,19 +16,19 @@
 
 package controllers
 
-import connectors.models.VatNoData
+import connectors.models._
 import controllers.actions._
 import controllers.helpers.AccountSummaryHelper
 import models.requests.AuthenticatedRequest
 import models.{VatDecEnrolment, VatNoEnrolment}
 import org.mockito.Matchers
 import org.mockito.Mockito.when
 import org.scalatest.mockito.MockitoSugar
-import play.api.mvc.AnyContent
+import play.api.mvc.{AnyContent, Result}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 import play.twirl.api.Html
-import services.VatService
+import services.{VatService, VatServiceInterface}
```

Comment:
Why have we added VatServiceInterface? Doesn't seem to be used.

Reply:
Ok. `VatServiceInterface` removed